mypyc/lib-rt/str_ops.c

```diff
@@ -201,4 +200,24 @@ Py_ssize_t CPyStr_Size_size_t(PyObject *str) {
         return PyUnicode_GET_LENGTH(str);
     }
     return -1;
-}
\ No newline at end of file
+}
+
+#define PyUnicode_UTF8(op)                              \
```

Comment:
It's better not use a macro here, as it buys us little and makes the code harder to understand and maintain. If you need to share code, use an inline function.